src/Centreon/Domain/Entity/Topology.php

```diff
@@ -99,6 +99,12 @@ class Topology
      */
     protected $topology_show;
 
+    /**
+     * @var string
+     * enum('0','1')
+     */
+    protected $is_deprecated;
```

Comment:
change the type to a boolean value and give it a default value (same as its database definition)